api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -493,12 +493,12 @@ message Features {
   bool SAML = 5 [(gogoproto.jsontag) = "saml"];
   // AccessControls enables FIPS access controls
   bool AccessControls = 6 [(gogoproto.jsontag) = "access_controls"];
-  // AdvancedAccessWorkflows enables advanced access workflows
-  // Deprecated: Use the [AccessRequests.Enabled] field instead.
```

Comment:
Ah, This is just a revert of https://github.com/gravitational/teleport/pull/34792 and it should be safe since it was not released.